notebooks/MIRI/Imaging/JWPipeNB-MIRI-imaging.ipynb

```diff
@@ -1467,7 +1467,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.12.9"
+   "version": "3.13.2"
```

Comment:
what's going on here with the notebook metadata? It's inconsistent with the other notebooks